eva/parser/parser.py

```diff
@@ -35,20 +35,20 @@ def syntaxError(self, recognizer, offendingSymbol, line, column, msg, e):
                     str(column) + " - " + str(msg)
         raise Exception(error_str)
 
-    def reportAmbiguity(self, recognizer, dfa, startIndex, stopIndex,
-                        exact, ambigAlts, configs):
-        error_str = "ERROR: Ambiguity -" + str(configs)
-        raise Exception(error_str)
+    # def reportAmbiguity(self, recognizer, dfa, startIndex, stopIndex,
```

Comment:
Can you share the reason for removing these?

Reply:
Given we are changing things, please also change the class name from MyErrorListener to something better maybe AntlrErrorListener

Reply:
I think our parser definition is incrementally defined over time by different people. There are duplication and ambiguity. 
This breaks almost all testcases involving parser.

Reply:
Ah! That is bad. How severe is it? Maybe create an issue and we can skip it for now.

Reply:
Create an issue: #264